src/cookie.js

```diff
@@ -70,3 +76,17 @@ function deleteCookie(name) {
     path: '/'
   })
 }
+
+if (typeof process !== 'undefined' &&
```

Comment:
????

Reply:
This condition detect node environment and make library available for testing